File: go/conditions.go
package main

import "fmt"

func main() {
	fmt.Println("SHALL WE PLAY A GAME?")
	var answer string
	fmt.Scanln(&answer)
	
	if answer == "yes" {
		fmt.Println("HOW ABOUT A NICE GAME OF CHESS?")
	} else {
		fmt.Println("OKAY THEN!")
	}
}

File: cpp/conditions.cpp
#include <iostream>
#include <string>

using namespace std;

int main() {
	string answer;

	cout << "SHALL WE PLAY A GAME?" << endl;
	getline(cin, answer);
	if (answer == "yes") {
		cout << "HOW ABOUT A NICE GAME OF CHESS?" << endl;	
	} else {
		cout << "OKAY THEN!" << endl;	
	}
	
	return 0;
}